runtime/common/src/lib.rs

```diff
@@ -52,17 +52,6 @@ macro_rules! decl_common_types {
 
         type Address = sp_runtime::MultiAddress<AccountId, ()>;
 
-        #[cfg(feature = "parachain")]
-        pub type Executive = frame_executive::Executive<
-            Runtime,
-            Block,
-            frame_system::ChainContext<Runtime>,
-            Runtime,
-            AllPalletsWithSystem,
-            zrml_prediction_markets::migrations::AddOutsiderBond<Runtime>,
-        >;
-
-        #[cfg(not(feature = "parachain"))]
```

Comment:
Don't we want the migrations from now on?

Reply:
[The migration is still included](https://github.com/zeitgeistpm/zeitgeist/pull/992/files/021803683fd5b1438e7b89a15406e7bc84383499#diff-3287a4ee060b1070a8e6559a094b4890a596a1b281effef54cafe59da20edf49L72), the distinction between the standalone and parachain environment was just removed to deduplicate the code, as it does not matter for the current migration.